src/content/en/2025/cookies.md

```diff
@@ -3,18 +3,646 @@
+        <td class="numeric">395</td>
+        <td class="numeric">365</td>
+        <td class="numeric">390</td>
+      </tr>
+      <tr>
+        <td>p90</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+      </tr>
+      <tr>
+        <td>p99</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+      </tr>
+      <tr>
+        <td>max</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Statistics for age of cookies set on desktop pages.") }}</figcaption>
+</figure>
+
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Age of cookies (mobile top 1M) in days </th>
+        <th>First-party</th>
+        <th>Third-party</th>
+        <th>All</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>min</td>
+        <td class="numeric">0</td>
+        <td class="numeric">0</td>
+        <td class="numeric">0</td>
+      </tr>
+      <tr>
+        <td>p25</td>
+        <td class="numeric">1</td>
+        <td class="numeric">30</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>median</td>
+        <td class="numeric">365</td>
+        <td class="numeric">270</td>
+        <td class="numeric">360</td>
+      </tr>
+      <tr>
+        <td>p75</td>
+        <td class="numeric">395</td>
+        <td class="numeric">365</td>
+        <td class="numeric">390</td>
+      </tr>
+      <tr>
+        <td>p90</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+      </tr>
+      <tr>
+        <td>p99</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+      </tr>
+      <tr>
+        <td>max</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Statistics for age of cookies set on mobile pages.") }}</figcaption>
+</figure>
+
+Cookies are set to an expiration date when they are created. If session cookies expire immediately after the session is over ([see previous section](#session)), most first- and third-party cookies do not and have a median age of a full year. The longer cookies live, the longer they can be used for re-identification or cross-site tracking which is why most tracking cookies are typically set to be stored in the browser for a longer time. The maximum age among the cookies that we can observe with the instrumentation and collection of the HTTP Archive Tools is of 400 days, this is aligned with the [hard limits](https://developer.chrome.com/blog/cookie-max-age-expires) that Chrome imposes on cookie `Expires` and `Max-Age` attribute.
```

Comment:
1. "The maximum age among the cookies that we can observe with the instrumentation and collection of the HTTP Archive Tools is of 400 days, this is aligned with the [hard limits](https://developer.chrome.com/blog/cookie-max-age-expires) that Chrome imposes on cookie `Expires` and `Max-Age` attribute." our measurement methodology likely either rejects or truncates higher dates.